clash-protocols-memmap/src/Protocols/MemoryMap/TypeDescription/TH.hs

```diff
@@ -0,0 +1,254 @@
+-- SPDX-FileCopyrightText: 2025 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE OverloadedRecordDot #-}
```

Comment:
Isn't this a default extension in the cabal file? If not, we do it everywhere else so it's fine to do that.